### DISCUSSION TITLE
Windows 8.1 anonymyzed Dnscrypt-proxy not loading some websites.

### DISCUSSION THREAD

**Discussion by neilishere** [DNSCrypt/dnscrypt-proxy#2511] [Q&A]
Hello, I cannot access my banking website on Windows after I installed this recently on 2 computers - also on mac os High sierra which works fine. Both installs have the same dnscrypt-proxy.toml file.
Here is the message:

"Error: "WWF2TUh4dnM=" - You're using an outdated browser or are using a Proxy service which is unsupported. Please update your browser or turn off your Proxy to continue, thank you."

I've tried several firefox browsers - 117, 115, 88, and ie11.  It worked for a few days after windows install, then was flagged or blocked by the bank.
All other logins work fine (webmail etc.)  - maybe not as hi security.
Again, the same 7 servers and relays config on both machines and not working on windows.
Do I need to download a config file or stamp to authorize access ?
Thank you for your help (and a great program ! )

**Comment by lifenjoiner** [comment]
Personal opinions:
1. domestic domains are not recommended to be resolved by DNS over seas.
2. resolving those special domestic domains by anonymyzed/overseas DNS is worse.

Have you tried [Forwarding](https://github.com/DNSCrypt/dnscrypt-proxy/wiki/Forwarding)?

**Reply by neilishere** [reply]
Thanks for your reply.
I don't think I understand what you mean by "domestic domains". I always thought that dns was for all intenet (WorldWideWeb - WWW). 
Anyway, the bank website I 'm trying to reach is in Chicago, US.  I am in the US.  The servers I am using are also in the US - mainly for speed.

I did try forwarding the bank websites to 9.9.9.9,8.8.8.8 , but that didn't work.  The log verified loading from forwarding-rules.txt.  The Mac system works fine without forwarding - just not Windows.

**Reply by lifenjoiner** [reply]
Just some guesses. I don't get much information from your descriptions.
I guessed DNS servers at different locations may get different IPs for the sites with CDN. That IP you connected to may be examined by the bank, together with your IP, and then concludes that IP should not be accessed by you.
Seems I missed, so forget it.

Provide more details, see if anyone can help.

**Reply by neilishere** [reply]
Yes, sure, I appreciate guesses also. New things to consider or try is always helpful.

Another development today : I tried a Firefox ESR browser on the Mac computer and it also gave the error on the bank homepage as on Windows.  This is without even logging in - so my ip must be flagged (both windows and mac computers use the same router).  I have a Pulse browser on the mac and the domain isn't blocked.  So maybe it's something in the firefox browsers? - which have a users.js file. The Pulse browser doesn't have a users.js.  So I need to look at the config entries of the firefox browsers and maybe I'll find something to do with proxies.
